content/en/docs/reference/using-api/cel.md

```diff
@@ -44,43 +44,160 @@ variables are available for that field.
+  <td>Validate the values of the 'details' map</td>
+</tr>
+</tbody>
+</table>
 
 ## CEL options, language features, and libraries
 
-CEL is configured with the following options, libraries and language features, introduced at the specified Kubernetes versions:
-
-| CEL option, library or language feature | Included | Availablity |
-|-----------------------------------------|----------|-------------|
-| [Standard macros](https://github.com/google/cel-spec/blob/v0.7.0/doc/langdef.md#macros) | `has`, `all`, `exists`, `exists_one`, `map`, `filter` | All Kubernetes versions |
-| [Standard functions](https://github.com/google/cel-spec/blob/master/doc/langdef.md#list-of-standard-definitions) | See [official list of standard definitions](https://github.com/google/cel-spec/blob/master/doc/langdef.md#list-of-standard-definitions) | All Kubernetes versions |
-| [Homogeneous Aggregate Literals](https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#HomogeneousAggregateLiterals) | | All Kubernetes versions |
-| [Default UTC Time Zone](https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#DefaultUTCTimeZone) | | All Kubernetes versions |
-| [Eagerly Validate Declarations](https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#EagerlyValidateDeclarations) | | All Kubernetes versions |
-| [Extended strings library](https://pkg.go.dev/github.com/google/cel-go/ext#Strings), Version 1 | `charAt`, `indexOf`, `lastIndexOf`, `lowerAscii`, `upperAscii`, `replace`, `split`, `join`, `substring`, `trim` | All Kubernetes versions |
-| Kubernetes list library | See [Kubernetes list library](#kubernetes-list-library) | All Kubernetes versions |
-| Kubernetes regex library | See [Kubernetes regex library](#kubernetes-regex-library) | All Kubernetes versions |
-| Kubernetes URL library | See [Kubernetes URL library](#kubernetes-url-library) | All Kubernetes versions |
-| Kubernetes authorizer library | See [Kubernetes authorizer library](#kubernetes-authorizer-library) | All Kubernetes versions |
-| Kubernetes quantity library | See [Kubernetes quantity library](#kubernetes-quantity-library) | Kubernetes versions 1.29+ |
-| CEL optional types | See [CEL optional types](https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#OptionalTypes) | Kubernetes versions 1.29+ |
-| CEL CrossTypeNumericComparisons | See [CEL CrossTypeNumericComparisons](https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#CrossTypeNumericComparisons) | Kubernetes versions 1.29+ |
+CEL is configured with the following options, libraries and language features,
+introduced at the specified Kubernetes versions:
+
+<table>
+<thead>
+<tr>
+  <th>CEL option, library or language feature</th>
+  <th>Included</th>
+  <th>Availability</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td>CEL option, library or language feature</td>
+  <td>Included</td>
+  <td>Availablity</td>
+</tr>
+<tr>
+  <td>[Standard macros](https://github.com/google/cel-spec/blob/v0.7.0/doc/langdef.md#macros)</td>
```

Comment:
```suggestion
  <td><a href="https://github.com/google/cel-spec/blob/v0.7.0/doc/langdef.md#macros">Standard macros</a></td>
```
Need to change all these markdown links to html style.

Reply:
OK.